jak-na-konflikty.md

```diff
@@ -0,0 +1,77 @@
+# Jak zvládat konflikty s kolegou?
+
+Cílem této příručky je pomoct ti zvládat konfliktní situace mezi tebou a ostatními Appliftery.
+
+V Kodexu Appliftera se můžeš dočíst, že řešení konfliktů je přirozenou součástí každé organizace. Proto je důležité, abys věděl, jak ke konfliktu přistupovat tak, abys minimalizoval jeho toxicitu pro tebe a pro tvého kolegu a zároveň aby tvé jednání směřovalo k jeho vyřešení.
+
+## Příprava
+
+Před tím, než začneš s kolegou konflikt řešit, je dobré si v hlavě či na papíře ujasnit pár věcí. 
+
+**Cílem přípravy není mít co nejvíc argumentů k tomu, abys svůj protějšek ubil argumenty.** Cílem je schopnost nahlédnout na danou věc z více úhlů, vžít se do svého kolegy a pochopit jeho motivaci a chování.
+
+- Co je věcným "odemočněným" předmětem konfliktu? Jak se daný problém dá popsat co nejvíc věcně?
+- Jak bude předmět konfliktu vnímat a vidět tvůj kolega?
+- Jaké kolem toho máš emoce? Jak se cítíš?
+- Jaké kolem toho bude mít emoce tvůj kolega? 
+- Jaké asi byly motivace kolegy? Proč postupoval tak jak postupoval?
+
+## Před konfliktem
+
+Pokud máš s nadcházejícím konfliktem spojených hodně negativních emocí, zkus je nejdříve zmírnit. Sedni si sám v tiché místnosti na 5 minut a zamedituj si o tvém kolegovi. Soustřeď se na jeho kladné vlastnosti a hezké věci, co jste spolu zažili. Uvědom si, že nacházející konflikt je pravděpodobně něco, čemu se za půl roku nad pivem zasmějete.
+
+## Vyvolání řešení konfliktu
+
+Zastav kolegu v soukromí a popros ho, jestli by si na tebe na chvilku neudělal čas. Řekni mu, že s ním potřebuješ vyřešit nějaké pnutí (klidně to můžeš říct přátelštější či vtipnější formou). Pokud čas nemá, domluv se na čase, kdy se to bude oběma stranám hodit.
+
+## Řešení konfliktu
+
+Konflikt řeš někde, kde budete mít soukromí a když na to budete mít oba dostatečné množství času.
+
+Postupuj volně dle následujícíh bodů. Popros kolegu aby ti naslouchal a nepřerušoval tě.
+
+1. **Tohle se stalo / Tohle se děje** => Popiš bez emocí předmět konfliktu. 
+2. **Takhle se kolem toho cítím** => Popiš jaký má na tebe předmět konfliktu emocionální dopad. Pokus se svoje emoce regulovat tak, aby to ten druhý ustál.
+3. **Potřeboval bych, aby se tohle stalo / změnilo** => Popiš, jakou očekáváš u kolegy změnu v jeho chování/jeho přístupu.
```

Comment:
Chybí mi tady ještě nějaký reasoning - proč si myslím, že je to špatně. Možná je to součástí bodu 2, ale moc to z toho necítím. Myslím, že to může být zpravidla hlavní část konfliktu, kdy si jeden myslí, že je to správně a nechápe, že se kolem toho ta druhá cítí špatně.